### ISSUE TITLE
Web accessibility check homepage

### ISSUE THREAD

**Issue by gfranzini** [acdh-oeaw/dig_ed_cat#69]
Hi, I'm in the process of checking the web app against the **IDI Web Accessibility WCAG 2.0 (Level AA)** checker (https://achecker.ca/) and it brings up only one known problem on the homepage, which is the insufficient colour contrast of the "explore" button. I tried looking for the correct .css file in the repository to make the correction myself but I can't find it. Could you replace the current shade of blue #5bc0de with the darker #1d5771 ?
Thank you.

With IDI, the check has to be done for every webpage, so if I find any other issues I'll report them as separate tickets.

**Comment by zxenia** [comment]
sure!

**Comment by gfranzini** [comment]
Fabulous, thanks! The same applies to the blue pop-up boxes in the edition pages that explain how to use the back/forward arrows. :-)
